task-manager/src/views/LoginView.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <v-container
+        fluid
+        class="d-flex align-center justify-center login-container"
+    >
+        <v-card class="elevation-12 login-card">
+            <v-card-title class="headline">Login</v-card-title>
+            <v-card-text>
+                <v-text-field
+                    v-model="username"
+                    label="Username"
+                    outlined
+                    dense
+                    required
+                ></v-text-field>
+                <v-text-field
+                    v-model="password"
+                    label="Password"
+                    type="password"
+                    outlined
+                    dense
+                    required
+                ></v-text-field>
+
+                Don't have an account?
+                <router-link to="/signup" color="primary"
+                    >Sign up here.</router-link
+                >
+            </v-card-text>
+            <v-card-actions>
+                <v-spacer></v-spacer>
+                <v-btn color="primary" @click="login">Login</v-btn>
+            </v-card-actions>
+        </v-card>
+    </v-container>
+</template>
+
+<script>
+export default {
+    data() {
+        return {
+            username: "",
+            password: "",
+        };
+    },
+    methods: {
+        async login() {
+            const success = await this.$store.dispatch("login", {
+                username: this.username,
+                password: this.password,
+            });
+
+            if (success) {
+                this.$router.push("/dashboard");
+            } else {
+                alert("Invalid username or password.");
```

Comment:
instead of just an alert. can we create a toast component? 
read what is toast messages the the errors should be red with white text and successs should be green with white text 
eg login successful. Or user does not exist

Reply:
I have added required changes